### PR TITLE
App summary

### PR DESCRIPTION
**Pull request by nwmac**

### CODE REVIEW

**Comment by richard-cox** [comment]
Few quick comments...
- Summary tab is labled as 'Build Info'.
- Horizontal shrink the window and the 'Cloud Foundry' section doesn't expand to the same width as `Application Info`.
- The 'Build Info' card looks like this is missing the 'stack' field. Could be a regression from elsewhere though. Also routes shows 0, but that might be tweaked when Irfan's PR goes in.
- Do we have an issue to track the population of content of the top row 3rd card?
- Sometimes fields like 'Modified' are empty. In those case we should supply some kind of default text to keep alignment looking grand
- Do we want a max width on the element containing the cards? Otherwise they stretch reeeeeal wide when the window is expanded
- We should switch the bottom row cards 'Build Info' and 'Deployment Info' around (the later would not often be populated)

File: src/app/features/applications/application/application-tabs-base/application-tabs-base.component.html [hunk 1]
```diff
@@ -23,114 +23,21 @@ <h1>{{ (applicationService.application$ | async)?.app.entity.name }} </h1>
           <mat-icon>stop</mat-icon>
         </button>
         <button mat-icon-button name="start" *ngIf="(applicationService.application$ | async)?.app.entity.state === 'STOPPED'" [disabled]="summaryDataChanging$ | async"
-          (click)="startApplication()">
+          (clicmediak)="startApplication()">
```
Comment (richard-cox) [inline src/app/features/applications/application/application-tabs-base/application-tabs-base.component.html]: typo

File: src/app/features/applications/application/application-tabs-base/tabs/build-tab/build-tab.component.ts [hunk 2]
```diff
@@ -21,6 +24,8 @@ export class BuildTabComponent implements OnInit {
 
   ngOnInit() {
 
+    // const { cfGuid, appGuid } = this.applicationService;
```
Comment (richard-cox) [inline src/app/features/applications/application/application-tabs-base/tabs/build-tab/build-tab.component.ts]: Comment not needed

File: src/app/features/applications/edit-application/edit-application.component.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+
+// import { UpdateApplication } from '../../../store/actions/application.actions';
+
+@Component({
+  selector: 'app-edit-application',
+  templateUrl: './edit-application.component.html',
+  styleUrls: ['./edit-application.component.scss']
+})
+export class EditApplicationComponent implements OnInit {
```
Comment (richard-cox) [inline src/app/features/applications/edit-application/edit-application.component.ts]: Strictly speaking, I think the OnInit/OnDestroy interfaces are only needed if we execute code in them.
There's a few other places where this applies (`ngOnInit() {}` and 
```
ngOnInit() {
}
```
)

File: src/app/shared/components/cards/custom-cards/card-app-instances/card-app-instances.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<mat-card class="card-app-instances">
```
Comment (richard-cox) [inline src/app/shared/components/cards/custom-cards/card-app-instances/card-app-instances.component.html]: We may want to refactor this folder. Previously it was for cards which live in lists and shown via the cards/cards.component. A place for custom-cards is good but I think we should have the two types of cards in separate places. TBH I used a bad naming convention when creating the folders for the list cards

**Comment by nwmac** [comment]
Addressed feedback!

**Pull request merged**